edelphi/src/main/webapp/_scripts/gui/panel/queryblockcontroller.js

```diff
@@ -426,6 +460,7 @@ Scale1DQueryPageController = Class.create(QueryPageController, {
     if (this._liveReportController)
       this._liveReportController.deinitialize();
   },
+  
   _initializeSlider: function (questionContainer) {
     this._sliderController = new QueryBlockScaleSliderFragmentController(questionContainer);
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Move the declaration of "QueryBlockScaleSliderFragmentController" before this usage. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)